controllers/twophase/types.go

```diff
@@ -128,6 +128,12 @@ func (r *Reconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		}
 
 		nextRecover := now.Add(*duration)
+		// TODO(at15): this should be validated in webhook instead of throwing error at runtime.
```

Comment:
Can you file an issue for this?

Reply:
Filed https://github.com/pingcap/chaos-mesh/issues/373